tests/rustdoc-gui/cursor.goml

```diff
@@ -8,6 +8,8 @@ assert-css: ("#toggle-all-docs", {"cursor": "pointer"})
 assert-css: ("#copy-path", {"cursor": "pointer"})
 
 // the search tabs
+click: "#search-button"
```

Comment:
I saw these two lines in a few different tests. Maybe put it into a function in `util.goml`?